metadata-io/src/main/java/com/linkedin/metadata/search/ranker/SearchRanker.java

```diff
@@ -32,7 +33,12 @@
    * Rank the input list of entities
    */
   public List<SearchEntity> rank(List<SearchEntity> originalList) {
-    return Streams.zip(originalList.stream(), fetchFeatures(originalList).stream(), this::updateFeatures)
+    List<SearchEntity> entitiesToRank = originalList;
+    if (!getFeatureExtractors().isEmpty()) {
```

Comment:
So we still cn invoke feature extractors? I believe these are useful in search snippets

Reply:
yes. this is for oss where I am removing all feature extractors since they are never used. There is no need to update the entities when there are no feature extractors

Reply:
OK